sia-task-hunter/src/main/test/com/sia/hunter/StringHelperTest.java

```diff
@@ -0,0 +1,356 @@
+package com.sia.hunter;
```

Comment:
Shouldn't package be `com.sia.hunter.helper`? (as in the source class)
If so, this would also apply to `OnlineTaskHelper`